ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/CompleteTaskStep.java

```diff
@@ -62,13 +65,15 @@ public void markAsCompleteOrFailed(
       final WorldDownloadState<NodeDataRequest> downloadState,
       final Task<NodeDataRequest> task) {
     if (task.getData().getData() != null) {
-      enqueueChildren(task, header, downloadState);
       completedRequestsCounter.inc();
       task.markCompleted();
       downloadState.checkCompletion(worldStateStorage, header);
     } else {
       retriedRequestsCounter.inc();
       task.markFailed();
+      if (failedCount.incrementAndGet() >= 100_000) {
```

Comment:
I think this can be configurable. In my opinion we already have a flag that we could maybe use

Reply:
This was a temporary counter, I think I am going to remove it altogether.